scm/driver/harness/webhook.go

```diff
@@ -248,3 +268,30 @@ func convertAction(src string) (action scm.Action) {
 		return
 	}
 }
+
+func convertPullReq(pr pullReq, ref ref, commit hookCommit) scm.PullRequest {
+	return scm.PullRequest{
+		Number: pr.Number,
+		Title:  pr.Title,
+		Closed: pr.State != "open",
+		Source: pr.SourceBranch,
+		Target: pr.TargetBranch,
+		Fork:   "fork",
```

Comment:
I assume this can be an empty string? not sure why thomas added it like this?

Reply:
I dont want to risk it with CI in this merge, can we do it later?

Reply:
sure!